jenkins-slaves/jenkins-slave-ansible/Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM openshift3/jenkins-slave-base-rhel7:latest
+FROM openshift/jenkins-slave-base-centos7
 
-USER root
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
 
-RUN yum install -y --setopt=tsflags=nodocs \
-      --disablerepo=* --enablerepo=rhel-7-server-rpms --enablerepo=rhel-7-server-extras-rpms \
-      https://releases.ansible.com/ansible/rpm/release/epel-7-x86_64/ansible-2.5.8-1.el7.ans.noarch.rpm && \
-    rpm -V ansible && \
-    yum clean all -y && \
-    rm -rf /var/cache/yum
+RUN INSTALL_PKGS="java-1.8.0-openjdk-devel.x86_64 git tree vim unzip curl" && \
+    x86_EXTRA_RPMS=$(if [ "$(uname -m)" == "x86_64" ]; then echo -n java-1.8.0-openjdk-devel.i686 ; fi) && \
+    yum install -y centos-release-scl-rh && \
+    yum install -y --enablerepo=centosplus $INSTALL_PKGS $x86_EXTRA_RPMS && \
+    yum clean all -y
+
+RUN yum install -y epel-release && \
+	yum install -y python-pip && \
+	pip install ansible==2.6.1
+
+# When bash is started non-interactively, to run a shell script, for example it
+# looks for this variable and source the content of this file. This will enable
+# the SCL for all scripts without need to do 'scl enable'.
+ADD contrib/bin/scl_enable /usr/local/bin/scl_enable
+ADD contrib/bin/configure-slave /usr/local/bin/configure-slave
+
+RUN chown -R 1001:0 $HOME && \
```

Comment:
Please merge this `RUN` with the others into one.

Reply:
@pabrahamsson I tried this and got `chmod: cannot access '/home/jenkins': No such file or directory` when doing a `docker build .`. 
I structured my Dockerfile the same way as https://github.com/openshift/jenkins/blob/master/agent-maven-3.5/Dockerfile
I can look into why this is the case if you would like me too, but I think that staying along the same structure as the `openshift/jenkins` will be good.

Reply:
@Gl4di4torRr that error makes no sense but for the sake of getting this PR merged let's move on.
Leave your Dockerfile as it is and we can get back to this later.